opennms-web-api/src/main/java/org/opennms/web/svclayer/SchedulerMessageSeverity.java

```diff
@@ -26,27 +26,12 @@
  *     http://www.opennms.com/
  *******************************************************************************/
 
-package org.opennms.web.report.database.model;
-
-import org.springframework.binding.validation.ValidationContext;
-
-/**
- * <p>TriggerDetailsValidator class.</p>
- *
- * @author ranger
- * @version $Id: $
- * @since 1.8.1
- */
-public class TriggerDetailsValidator {
-    
-    /**
-     * <p>ValidateTriggerDetails</p>
-     *
-     * @param triggerDetails a {@link org.opennms.web.report.database.model.TriggerDetails} object.
-     * @param context a {@link org.springframework.binding.validation.ValidationContext} object.
-     */
-    public void ValidateTriggerDetails(TriggerDetails triggerDetails, ValidationContext context) {
-        //TODO validate the cron expression here
-    }
+package org.opennms.web.svclayer;
 
+// TODO MVR delete me
+public enum SchedulerMessageSeverity {
```

Comment:
The backend will be reworked at a later and with that these classes will be removed as well. As the focus of this PR is to remove spring-webflow the clasess `SchedulerMessageSeverity`, `SchedulerContext` and `SchedulerMessage` were created as a temporary replacement for the old `(Spring)Context`.